ses

```diff
@@ -130,6 +130,25 @@ if [ -d /etc/salt ]; then
 fi
 
 #############################################################
+section_header "NFS-Ganesha config"
+
+if [ -e /etc/ganesha/ganesha.conf ]; then
+    mkdir -p $LOG/ceph/ganesha/
+    cp /etc/ganesha/ganesha.conf $LOG/ceph/ganesha/ganesha.conf
+    plugin_message "NFS-Ganesha config copied to ceph/ganesha/ subdirectory"
+fi
+
+#############################################################
+section_header "NFS-Ganesha log files"
+
+if [ -e /var/log/ganesha.log ]; then
+    mkdir -p $LOG/ceph/ganesha/log
+    cp /var/log/ganesha.log $LOG/ceph/ganesha/log
```

Comment:
Can you change this so it creates and copies to `$LOG/ceph/log/ganesha` instead of `$LOG/ceph/ganesha/log`? That way all the log files for all the daemons will be under `$LOG/ceph/log/...` (we've already got `$LOG/ceph/log/ceph/`, `$LOG/ceph/log/calamari/`, etc., following that pattern).